README.md

```diff
@@ -7,10 +7,143 @@
 
 ## Overview
 
-The µONOS Topology subsystem provides topology management for µONOS core services and applications. The topology is exposed to services via an implementation of the onos-topo gRPC API as defined by the [onos-api]. The toplogy API is generalized as a set of entities and relations, where `Entity` models objects within the system (e.g. devices, nodes, etc) and `Relation` models the associations between objects.
+The µONOS Topology subsystem provides topology management for µONOS core services and applications.
+The topology subsystem structures the topology information as a set of objects, which can be either 
+`Entity`, `Relation` or a `Kind`.
 
-The topology subsystem is shipped as a [Docker] image and deployed with [Helm]. To build the Docker image, run `make images`.
+* `Entity` objects are nodes in a graph and are generally intended to represent network devices, control entities, 
+control domains, and so on.
+* `Relation` objects are edges in a graph and are intended to represent various types of relations between two 
+`Entity` objects, e.g. `contains`, `controls`, `implements`.
+* `Kind` objects can be thought of as template or schema objects that represent an entity or a relation kind. 
+Strictly speaking, `Entity` or `Relation` instances do not have to be associated with a `Kind`, 
+but maintaining `Kind` associations can be used for schema validation and speeding up queries and is 
+therefore highly encouraged.
 
-[onos-api]: https://github.com/onosproject/onos-api
+## API
```

Comment:
I think we should explain RAN entities, their relations, aspects  as well as a use case of EKR model. Perhaps somewhere under docs, we create a document for RAN use case. What do you think @tomikazi ?

Reply:
Yes, I agree.